### ISSUE TITLE
Glist ban broadcasts last login name rather than name specified

### ISSUE THREAD

**Issue by Obi2** [TotalFreedom/TotalFreedomMod#763]
Using glist ban on a player broadcasts the last username which has logged in under the player's IP. This behavior may be intended, but it is not particularly useful when banning players which have logged in as administrators, adding to confusion.

**Comment by PattyToTheWatty** [comment]
Yes plz. Its so annoying!

**Comment by ValenciaOrange** [comment]
The name that you are trying to ban is a alt. the name that it broadcast and bans  is the real account. I believe it's is because they have the same uuid

**Comment by Obi2** [comment]
@ValenciaOrange Yes, I'm saying that this behavior is often unwanted in the case that you and I described.

**Comment by PacksGamingHD** [comment]
Yes!
